ChrootBuild.sh

```diff
@@ -59,14 +66,29 @@ function UsageMsg {
 }
 
 function PrepChroot() {
-   local REPOPKGS=($(echo \
-                     "$(rpm --qf '%{name}\n' -qf /etc/redhat-release)" ; \
-                     echo "$(rpm --qf '%{name}\n' -qf \
-                            /etc/yum.repos.d/* 2>&1 | \
-                            grep -v "not owned" | sort -u)" ; \
-                     echo yum-utils
-                   ))
 
+   if [ -f "/etc/oracle-release" ]
```

Comment:
Unfortunately, our CI system is not currently set up to be able to validate this. I'll have to assume that it's basically correct* as written. That said...